drivers/display/ssd16xx.c

```diff
@@ -105,6 +112,8 @@ struct ssd16xx_config {
 	uint16_t height;
 	uint16_t width;
 	uint8_t tssv;
+	uint8_t gdo_flags;
+	bool scan_y_reverse;
```

Comment:
Why is this property needed? Is there a specific use-case that requires it?

Reply:
Yes, as in the example, the Waveshare 4.26" 800x480 display needs. As with the default scan modes, the displayed image is flipped. We need to use the data entry modes that has the Y direction reversed for the display to be correct.